src/com/student/Main.java

```diff
@@ -1,8 +1,40 @@
 package com.student;
 
+import java.io.*;
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args) {
+        System.out.println("Usage: search <pattern> <full path to file>");
+        String command = "";
+        while (!command.equals("exit")) {
```

Comment:
Where does exit come in? Maybe use .hasNextLine() + do the first read before the while?

Reply:
That makes more sense than introducing a new command, yes. Will change.